examples/poiseuille-mpi.py

```diff
@@ -478,7 +486,7 @@ def my_rhs(t, state):
     parser.add_argument("--casename", help="casename to use for i/o")
     args = parser.parse_args()
 
-    lazy = args.lazy
+    lazy = args.lazy or args.esdg
```

Comment:
This behavior could be surprising. Maybe just raise an exception if `lazy=False` and `esdg=True`?

Reply:
When using ESDG, I, @tulioricci, and @hyperSuperCube have struggled with consistently setting the correct options.  

Since ESDG will not run without lazy, and will not run without overintegration, to me it makes sense to just automatically enable those when ESDG is used.  It's also a pain when my batch job fails because I failed to specify all the required options to support ESDG.   I prefer to keep this - but I am OK with removing if you absolutely hate it.

Reply:
Yeah, I guess a warning would be good enough.